src/opcode.rs

```diff
@@ -1447,15 +1447,14 @@ opcode! {
         let MsgRingData { ring_fd, result, user_data, user_flags, opcode_flags } = self;
 
         let mut sqe = sqe_zeroed();
-        sqe.opcode = Self::CODE;
-        sqe.__bindgen_anon_2.addr = sys::IORING_MSG_DATA.into();
+        sqe.opcode = Self::CODE; sqe.__bindgen_anon_2.addr = sys::IORING_MSG_DATA.into();
```

Comment:
There seems to be a line break missing here.

Reply:
Sorry. It seems like RustRover has some formatting problems in that file.
It removed all line breaks in my previous commit as well and I forgot to check this time.